src/components/pages/overview-page/OverviewPage.module.scss

```diff
@@ -12,24 +10,25 @@ $situationPageSectionMaxWidth: 37.5rem;
         flex-direction: column;
         justify-content: flex-start;
         margin-top: 2rem;
-        max-width: $situationPageSectionMaxWidth;
+        max-width: 56.25rem;
         width: 100%;
+
+        :global(.layout) {
+            max-width: 37.5rem;
+        }
     }
 
     .productListWrapper {
         background-color: white;
         padding: 2rem;
+        max-width: 37.5rem;
```

Comment:
```suggestion
        max-width: common.$sectionMaxWidth;
```

Reply:
Disse finnes vel bare i less-to-scss-complete, men jeg kan godt definere de inn i common i min branch og så konfliktløser vi heller senere!